tests/conftest.py

```diff
@@ -4,17 +4,15 @@
 
 import pytest
 import yaml
-
+from conda_build.utils import copy_into
 from jinja2 import FileSystemLoader
 from jinja2.sandbox import SandboxedEnvironment
```

Comment:
Think the idea here was to group `import`s and then `from...import...`s and separate those out

Does `ruff` have a way to do this?

Reply:
From what I've seen, black/isort distinguish three blocks (one blank line of separation):

- stdlib
- 3rd party
- the package itself

Then within each block, we'll see the `import` lines first, followed by `from X import Y`, alphabetically. No blank line.

Reply:
I also think so. The reasoning behind `isort` makes sense and we don't need to change something about it IMO.